src/Microsoft.DotNet.Build.Tasks/PackageFiles/msbuild.cmd

```diff
@@ -1,2 +1,2 @@
-@call msbuild.exe %*
+@call %~dp0dotnetcli\dotnet.exe msbuild %*
```

Comment:
We probably want to add some logic to see if msbuild is on the path and if so then use it otherwise fallback to dotnet.exe msbuild. With some way to force that if needed.

Reply:
I don't think keying off the `%PATH%` is a good idea.  If I happen to use a VS developer command prompt and build this repo, by default I think it should build the same as if I opened a blank `cmd` prompt.

Instead, I think the `force` option should be the other way around.  By default, use the `dotnet.exe msbuild`, and keep it consistent between Windows and Linux.  If someone wants to force the build to use a full framework MSBuild, then they can pass in an arg to override the MSBuild to use.

Reply:
I believe our only concern on using the .NETCore MSBuild that comes with the CLI as the default was perf, since the desktop version was usually faster. Once we are ready to push this through, we can try and measure perf again with each case, and then decide on which one to take based on both consistency across platforms and build time.

Reply:
If we have concerns about perf - then our customers will too.  We need to push on getting these things into our product.  That's the point about dogfooding.